chains/evm/config_test.go

```diff
@@ -89,53 +84,50 @@ func TestChainConfigOneContract(t *testing.T) {
 
 func TestRequiredOpts(t *testing.T) {
 	// No opts provided
-	input := chains.RawChainConfig{
-		Opts: map[string]string{},
-	}
+	input := RawEVMConfig{}
 
-	_, err := ParseConfig(&input)
+	_, err := parseConfig(&input)
 
 	if err == nil {
 		t.Error("config missing chainId field but no error reported")
 	}
 
 	// Empty bridgeContract provided
-	input = chains.RawChainConfig{
-		Opts: map[string]string{"bridge": ""},
-	}
+	input = RawEVMConfig{Bridge: ""}
 
-	_, err2 := ParseConfig(&input)
+	_, err2 := parseConfig(&input)
```

Comment:
Use the same `err` variable instead of initializing + declaring a new one.

So, since `err` has already been declared, just set it to the new value

```go
_, err = parseConfig(&input)
```